packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -104,6 +105,9 @@ const OutlineLink = styled(RouterButton).attrs({
   border: 1px solid ${({ theme }) => theme.palette.primary.main};
   color: ${({ theme }) => theme.palette.primary.main};
   background: transparent;
+  text-transform: none;
+  line-height: 20px;
```

Comment:
Line height is usually done in decimal values of the font size, e.g. `line-height: 1.4;`